public/js/server.js

```diff
@@ -254,22 +246,15 @@ Server.deleteArchive = function (arcId, callback) {
         .then((response) => (response.ok ? response.json() : { success: 0, error: "Response was not OK" }))
         .then((data) => {
             if (data.success === "0") {
-                $.toast({
-                    showHideTransition: "slide",
-                    position: "top-left",
-                    loader: false,
+                window.toast({
                     heading: "Couldn't delete archive file. <br> (Maybe it has already been deleted beforehand?)",
                     text: "Archive metadata has been deleted properly. <br> Please delete the file manually before returning to Library View.",
-                    hideAfter: false,
```

Comment:
hideAfter and autoClose are both OK due to the "compatability layer" I wrote.
It's your decision if these toastes should be auto closed.
I removed `hideAfter: false`, so that toasts can be closed in default timer. (5s for normal info and 7s for warning and errors. In both situation, lose focus or hover the mouse on the toast can pause the timer.)

Reply:
Personally I don't want these warnings stay in the page all the time.

Reply:
Most of those aren't warnings but full-on errors, which I sincerely believe shouldn't dismiss on their own as we must make sure the user gets to see them.  
Warnings dismissing after a longer time is OK.  

The remaining toasts that used `hideAfter: false` are the help ones since they're a bit long to read -- For this the hover-on-toast to pause the timer behavior should indeed be enough.

Reply:
Yep. The toast now follows this rules:
success: close after 5s
info: close after 5s
warning: close after 10s
error: never auto close

All the toast are read by myself, and the reading time is measured using a timer, as long as possible.

We also provide a fallback if mouse hovers on the toast. (Not sure if it is intuitive enough, but it is now treated as a fallback)

Besides, it is possible to give every toast a unique ID, so they won't duplicate, and if error keeps occuring, when one toast with its unique ID disappear, a same one can be generated to make it always on screen. This needs tons of work, and its not implement yet, but possible.